src/Microsoft.DotNet.Wpf/src/WpfGfx/core/common/renderoptions.cpp

```diff
@@ -41,6 +48,7 @@ HRESULT
 RenderOptions::Init()
 {
     m_fForceSoftware = false;
+    m_fHwAccelerationForRDPEnabled = false;
```

Comment:
shouldn't we make this opt out and default to true?

Reply:
see [comment](https://github.com/dotnet/wpf/pull/7684#issuecomment-1628405496)